draft-packages/filter-menu-button/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@kaizen/draft-filter-menu-button",
+  "version": "1.0.0",
+  "description": "The draft filter-menu-button component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '*.d.ts' '*.js' '*.map' 'KaizenDraft/**/*.d.ts' 'KaizenDraft/**/*.js' 'KaizenDraft/**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "draft-packages/filter-menu-button"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {
+    "@kaizen/component-library": "^8.0.6",
+    "@kaizen/draft-button": "^3.0.6",
```

Comment:
As far as I can tell this isn't actually a dependency, so this can be removed. It’s used in the story, but the dependencies for stories are managed separately.

Reply:
I still use the secondary button from kaizen draft button here https://github.com/cultureamp/kaizen-design-system/pull/893/files#diff-4f1fc9a720a04a16455d3077a4de2ddce142d74afe58b9e975626b13af035915R61.

Even though it hasn't has the state that it needs as I've discussed [here](https://cultureamp.slack.com/archives/C2ZNME08P/p1607579632122100) on slack

Reply:
Ah, sorry I missed that.